src/guides/v2.4/release-notes/release-notes-2-4-0-open-source.md

```diff
@@ -1462,24 +1491,24 @@ We have fixed hundreds of issues in the Magento 2.4.0 core code.
 
 <!--- ENGCOM-6809-->
 
-*  The `getCustomer` method now returns the customer object as defined in `phpdoc` block. Previously, this method returned `NULL`, and `phpdoc` block did not highlight that this method could could return `NULL`. _Fix submitted by Fanis Strezos in pull request [26423](https://github.com/magento/magento2/pull/26423)_. [GitHub-25268](https://github.com/magento/magento2/issues/25268)
+*  The `getCustomer` method now returns the customer object as defined in `phpdoc` block. Previously, this method returned `NULL`, and `phpdoc` block did not highlight that this method could return `NULL`. _Fix submitted by Fanis Strezos in pull request [26423](https://github.com/magento/magento2/pull/26423)_. [GitHub-25268](https://github.com/magento/magento2/issues/25268)
+
+<!--- ENGCOM-6492-->
+
+*  Re-orders are now disabled as expected when a merchant disables the  **Admin** > **Store**  >  **Configuration**  >  **Sales**  >  **Reorder**  >  **Allow Order**.  _Fix submitted by Eden Duong in pull request [26051](https://github.com/magento/magento2/pull/26051)_. [GitHub-25130](https://github.com/magento/magento2/issues/25130)
 
 <!--- ENGCOM-6627-->
 
 *  `getPrice()` method now returns a float or null instead of a string when setting a custom price for an order in the Admin.  _Fix submitted by Andrii  Chorniy in pull request [26313](https://github.com/magento/magento2/pull/26313)_. [GitHub-25968](https://github.com/magento/magento2/issues/25968)
 
 <!--- ENGCOM-6786-->
 
-*  The **State Code and Title** column of **Admin** > **Stores**   >  **Settings**   >  **Order Status** now displays state instead of status as expected.  _Fix submitted by Oleh Usik in pull request [26569](https://github.com/magento/magento2/pull/26569)_. [GitHub-17847](https://github.com/magento/magento2/issues/17847)
+*  The **State Code and Title** column of **Admin** > **Stores**  > **Settings**  >  **Order Status** now displays state instead of status as expected.  _Fix submitted by Oleh Usik in pull request [26569](https://github.com/magento/magento2/pull/26569)_. [GitHub-17847](https://github.com/magento/magento2/issues/17847)
```

Comment:
```suggestion
*  The **State Code and Title** column of  **Admin** > **Stores**  > Settings >  **Order Status** now displays state instead of status as expected.  _Fix submitted by Oleh Usik in pull request [26569](https://github.com/magento/magento2/pull/26569)_. [GitHub-17847](https://github.com/magento/magento2/issues/17847)
```

Reply:
Removed formatting as suggested